### PR TITLE
Updating for TShock 4.5.3+4

### PR DESCRIPTION
**Pull request by csmir**
Changing TShock.Config to TShock.Config.Settings in DataHandlers.cs > Tested on TShock 4.5.3 & 4

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\Dimensions.sln",
+  "PreviewInSolutionExplorer": false
+}
```
Comment (popstarfreas) [inline .vs/VSWorkspaceState.json]: What is this file?

File: Dimensions/Dimensions.csproj [hunk 2]
```diff
@@ -59,4 +56,4 @@
     <Compile Include="DataHandlers.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-</Project>
+</Project>
```
Comment (popstarfreas) [inline Dimensions/Dimensions.csproj:62]: For the references, you need to update the ones in the Refs folder instead of one in a path outside the project.

**Review by popstarfreas — CHANGES_REQUESTED** [review]
Thanks for the PR.

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (csmir) [reply .vs/VSWorkspaceState.json]: It makes it so the solution can be opened in preview before opening it in the solution. This file can be removed, but was pushed alongside the commit.

**Review by popstarfreas — DISMISSED** [review]

**Comment by popstarfreas** [comment]
It says the TShockAPI.dll Ref version is 4.5.2.0. Can you put in the latest one?

**Pull request closed without merging**